pstress/pstress-run.sh

```diff
@@ -1389,18 +1389,15 @@ elif [ "${VERSION_INFO}" != "5.7" -a "${VERSION_INFO}" != "8.0" ]; then
 fi
 
 if [[ ${PXC} -eq 0 && ${GRP_RPL} -eq 0 ]]; then
```

Comment:
if it is PXC, where do we copy these files (mysqld, lib dir etc)?

Reply:
check line no. 1434 onwards

Reply:
ah right!